core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadOnlyLDAPUserStoreManager.java

```diff
@@ -2521,7 +2520,6 @@ public boolean doCheckIsUserInRole(String userName, String roleName) throws User
                     LdapName ldn = new LdapName(nameInSpace);
                     membershipValue = escapeLdapNameForFilter(ldn);
                 } catch (InvalidNameException e) {
-                    log.error("Error while creating LDAP name from: " + nameInSpace);
                     throw new UserStoreException("Invalid naming exception for : " + nameInSpace, e);
```

Comment:
Removed unnecessary space in https://github.com/wso2/carbon4-kernel/pull/383/files#diff-1a26e09d32d7214558ad5e3e663e9b6eR1917